### ISSUE TITLE
A11y - OU: Table header text should not be empty

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5474]
Ensure table headers have discernible text. [https://dequeuniversity.com/rules/axe/4.10/empty-table-header?application=AxeChrome|https://dequeuniversity.com/rules/axe/4.10/empty-table-header?application=AxeChrome|smart-link] 

h3. Element Location:

{noformat}.b2b-table-action-column-actions.b2b-table-column[scope="col"]{noformat}

h3. To solve this problem, you need to fix the following:

Element does not have text that is visible to screen readers

!Screen Shot 2024-11-13 at 16.38.22 PM.png|width=1520,height=734,alt="Screen Shot 2024-11-13 at 16.38.22 PM.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39574**



[created from NEXT-39574, comment 512399]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.0**

 This issue will be / was solved in Shopware version ***6.6.10.0***.



[created from NEXT-39574, comment 519706]
